README.md

```diff
@@ -2,334 +2,34 @@
-[![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://discord.gg/tsBTgc9)
-[![Greenkeeper badge](https://badges.greenkeeper.io/sourcecred/sourcecred.svg)](https://greenkeeper.io/)
+[![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://sourcecred.io/discord)
 
-SourceCred creates reputation networks for open-source projects.
-Any open-source project can create its own _cred_, which is a reputational metric showing how much credit contributors deserve for helping the project.
-To compute cred, we organize a project’s contributions into a graph, whose edges connect contributions to each other and to contributors.
-We then run PageRank on that graph.
+SourceCred allows communities to assign Cred scores, which measure the value that contributors have brought to the community.
+The community can then use the scores to distribute rewards (e.g. project-specific Grain tokens) to their contributors.
 
-To learn more about SourceCred’s vision and values, please check out [our website] and our [forum].
-One good forum post to start with is [A Gentle Introduction to Cred].
+Cred is computed by constructing a Cred Graph showing every contribution to a project, and then running a version of the PageRank
+algorithm to assign scores to every contribution and contributor.
 
-For an example of SourceCred in action, you can see SourceCred’s own [prototype cred attribution][prototype].
+We currently support loading data from GitHub, Discourse, Discord, and via custom "initiatives".
+
+Please check out [our website] for more information. If you'd like to get involved as a contributor, please drop by [our Discord]
+and say "hi"!
 
 [our website]: https://sourcecred.io/
-[prototype]: https://sourcecred.io/cred/timeline/@sourcecred/
-[A Gentle Introduction to Cred]: https://discourse.sourcecred.io/t/a-gentle-introduction-to-cred/20
+[our Discord]: https://sourcecred.io/discord
 
 ## Current Status
 
-We have a [prototype] that can generate a cred attribution based on GitHub interactions (issues, pull requests, comments, references, etc.).
-We’re working on adding more information to the prototype, such as tracking modifications to individual files, source-code analysis, GitHub reactions, and more.
-
-### Running the Prototype
-
-If you’d like to try it out, you can run a local copy of SourceCred as follows.
-First, make sure that you have the following dependencies:
-
-  - Install [Node] (tested on v12.x.x and v10.x.x).
-  - Install [Yarn] (tested on v1.7.0).
-  - Create a [GitHub API token]. No special permissions are required.
-  - For macOS users: Ensure that your environment provides GNU
-    coreutils. [See this comment for details about what, how, and
-    why.][macos-gnu]
-
-[Node]: https://nodejs.org/en/
-[Yarn]: https://yarnpkg.com/lang/en/
-[GitHub API token]: https://github.com/settings/tokens
-[macos-gnu]: https://github.com/sourcecred/sourcecred/issues/698#issuecomment-417202213
```

Comment:
I feel like some of this stuff might still be relevant to new contributors and people exploring. Maybe we should move it to a temporary separate MD file and link to it from the main README for developers with a disclaimer that it might contain outdated info. Will likely also serve as a good reference when writing the new replacement docs as well.

Reply:
Good point. I kept around these dev instructions (they'll stay relevant) but left out the stuff for actually turning on the dev server, since that won't work anymore.